src/AWS.Deploy.Common/ProjectDefinitionParser.cs

```diff
@@ -126,7 +126,9 @@ private async Task<bool> ValidateProjectInSolution(string projectPath, string so
 
             var lines = await _fileManager.ReadAllLinesAsync(solutionFile);
             var projectLines = lines.Where(x => x.StartsWith("Project"));
-            var projectPaths = projectLines.Select(x => x.Split(',')[1].Replace('\"', ' ').Trim()).ToList();
+            var projectPaths = projectLines.Select(x => x.Split(',')[1].Replace('\"', ' ').Trim())
+                .Select(x => x.Replace('\\', Path.DirectorySeparatorChar))
+                .ToList();
```

Comment:
We should remove this method than fixing  and start using https://github.com/aws/aws-dotnet-deploy/blob/cc7cbcffb353d41aa2c73ead71468d66c80ab72c/src/AWS.Deploy.Orchestration/DeploymentBundleHandler.cs#L195

Reply:
This method is redundant. So removed it.